src/ORM/DB.php

```diff
@@ -82,7 +129,7 @@ class DB
      * be accessed through DB::get_conn($name).  This is useful when you have an application that
      * needs to connect to more than one database.
      */
-    public static function set_conn(Database $connection, $name = 'default')
+    public static function set_conn(Database $connection, $name = DB::CONN_DEFAULT)
```

Comment:
I won't change the method signature because this is for a minor release

Throwing an exception seems a little rough

I've opted to simply do this, which we're doing for things like DB::setConfig()

```php
        if ($name === DB::CONN_DEFAULT) {
            $name = DB::CONN_PRIMARY;
        }
```